src/graphql-global-id/package.json

```diff
@@ -19,6 +19,6 @@
     "@babel/runtime": "^7.17.2"
   },
   "peerDependencies": {
-    "graphql": "^15.0.0"
+    "graphql": "^16.0.0"
```

Comment:
What about something like `^15.0.0 || ^16.0.0` to unblock it for version 16? There are still some projects in Universe that depend on this package and they are not upgraded to the latest GraphQL version.